src/processing/ParameterizedOperationsUtil.java

```diff
@@ -3,26 +3,40 @@
 import java.util.*;
 import processing.boardobject.*;
 import processing.utility.*;
+import infrastructure.validation.logger.*;
 
 /**
  *
  * @author Devansh Singh Rathore
  * @reviewer Shruti Umat
  */
 
+
 /**
  * Class for implementing selected(non-erase, non-reset) object's Color-change and Rotate
  * operation.
  */
 public class ParameterizedOperationsUtil {
 
+    // initialise a reference to the logger singleton object
+	ILogger paraOpLogger = LoggerFactory.getLogger();
```

Comment:
Logger Factory does not have a `getLogger`method. Its `getLoggerInstance`.

Reply:
done. thanks.